src/main/groovy/life/qbic/datamodel/dtos/business/facilities/Facility.groovy

```diff
@@ -17,22 +17,35 @@ package life.qbic.datamodel.dtos.business.facilities
  */
 enum Facility {
 
-    CFMB("Core Facility for Medical Bioanalytics"),
-    IMGAG("Institute for Medical Genetics and Applied Genomics"),
-    MGM("Institute for Medical Microbiology and Hygiene"),
-    QBIC("Quantitative Biology Center"),
-    PCT("Proteome Center Tübingen")
+    CFMB("Core Facility for Medical Bioanalytics", "CFMB"),
+    IMGAG("Institute for Medical Genetics and Applied Genomics", "IMGAG"),
+    MGM("Institute for Medical Microbiology and Hygiene", "MGM"),
+    QBIC("Quantitative Biology Center", "QBIC"),
+    CFMB_PCT("Proteomics Facility Tübingen", "Proteomics Facility")
 
     private final String fullName
+    private final String label
 
     /**
      * Creates an instance of a facility enum
-     * @param fullName The full name representation of the enum
+     * @param fullName The full name representation of the facility
      */
     Facility(String fullName) {
         this.fullName = fullName
+        this.label = ""
     }
 
+    /**
+     * Creates an instance of a facility enum with shorthand label
+     * @param fullName The full name representation of the facility
+     * @param label The shorthand label of the facility
+     */
+    Facility(String fullName, String label) {
+        this(fullName)
```

Comment:
You can replace this constructor chaining by removing the old one and integrating the one line here instead
```suggestion
        this.fullName = fullName
```